packages/system/src/Item/api/hooks/useDefaultServiceItem/useDefaultServiceItem.tsx

```diff
@@ -0,0 +1,11 @@
+import { useServiceItems } from '../useServiceItems';
+
+export const useDefaultServiceItem = () => {
+  const { data: items, isLoading, error } = useServiceItems();
+
+  const defaultServiceItem = items?.nodes.find(
+    ({ code }) => code === 'service'
+  );
```

Comment:
TBH This isn't ideal. I was wondering if some explicit queries for the 'default' service item/non stock item etc to be sure 🤷

Reply:
yeah, I'd prefer to have the 'default' status set explicitly, but there's nothing in mSupply for that. This will do for now